frontend/sw360-portlet/src/main/webapp/js/modules/sw360Wizard.js

```diff
@@ -0,0 +1,244 @@
+/*
+ * Copyright Siemens AG, 2017.
+ * Part of the SW360 Portal Project.
+ *
+ * SPDX-License-Identifier: EPL-1.0
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+define('modules/sw360Wizard', [ 'jquery' ], function($) {
+
+    /**
+     * This module provides a wizardination of some given html, specialized on doing backend calls after each
+     * step to receive data for the content of the next step. And here is what you have to do:
+     *
+     * 1. Write some markup describing the steps of your wizard
+     * <div id="myWizard" data-step-id="0" data-foo="bar">
+     *     <div class="wizardHeader">
+     *         <!-- define your list of step headers -->
+     *         <ul>
+     *             <li class="active">1. Step Name<br /><small>This can be a short description</small></li>
+     *             <li>2. Another Step Name<br /><small>Another description</small></li>
+     *         </ul>
+     *     </div>
+     *     <div class="wizardBody">
+     *         <!-- define your list of steps -->
+     *         <div class="step active" data-step-id="1">
+     *             Default content of step 1 before response is received
+     *         </div>
+     *         <div class="step" data-step-id="2">
+     *             Default content of step 2 before response is received
+     *         </div>
+     *     </div>
+     * </div>
+     *
+     * 2. You can then grab the wizard framework via require and configure it inside like that:
+     * require([ 'modules/sw360Wizard', 'jquery' ], function(wizard, $) {
+     *     wizard({
+     *         wizardRoot: $('#myWizard'),
+     *         postUrl: your-post-url-goes-here,
+     *         postParamsPrefix: your-post-params-prefix-goes-here,
```

Comment:
I think calling it something like 'httpPostParamsPrefix' would be a more direct hint at what this parameter does. 
Although it is explained well enough in the long text below, so renaming is really optional.